Composer/packages/client/src/components/Pagination.tsx

```diff
@@ -45,6 +45,10 @@ const createDropdownOption = (pageCount: number) => {
   }));
 };
 
+// Note: these should be made RTL-aware at some point
+const PREVIOUS_LABEL = '← ' + formatMessage('Previous');
```

Comment:
Could we use an icon in the button instead?

Reply:
Changed this to use icons. Pending comment from design: 
![image](https://user-images.githubusercontent.com/61990921/88316307-7665d000-cccc-11ea-819c-20bf56b04f79.png)